docs/contributor/yourls-action/README.md

```diff
@@ -0,0 +1,108 @@
+# YOURLS Actions Quickstart
+
+This document serves to familiarize the *Contributor* with YOURLS Action by showing how to build and run it from source in a local build environment (*Environment*). This software populates the YOURLS database from a filesystem of CSV mapping files. This is used to load the [Namespaces](/namespaces/) directory to a Yourls compliant, MySQL database.
+
+
+# Prerequisites
+
+In order to build YOURLS Action you must have the following installed in your *Environment*. 
+
+## Python
+Python can be downloaded and installed from the official [Python website](https://python.org/). Ensure you select the appropriate version for your operating system.
+
+As of the last update, the YOURLS Action requires Python 3.6 or later. Check the system's documentation or codebase for the specific version requirements.
+
+This documentation may refer to the ``PYTHONPATH`` environment variable, which indicates the directories where Python looks for modules. If not set, Python will use a default path.
+
+Please refer to the Python official documentation for comprehensive information on Python installation, configuration, and development practices.
+
+```bash
+python3 --version
+pip3 --version
+```
+
+## Docker
+Docker is used here to launch YOURLS developer runtime service dependencies YOURLS and its database (MySQL). 
+
+# Building YOURLS Action
+
+Choose a starting location to work on your computer:
+
+```bash
+export SRC_BASE_DIR=/path/to/dev/directory
+```
+
+## Clone
+Clone from your forked github repository to your *Environment* in a predefined directory location.
+
+```bash
+cd $SRC_BASE_DIR
+git clone https://github.com/cgs-earth/yourls-action.git
+```
+
+## Runtime Dependencies
+
+### Set up YOURLS environment:
+To build the YOURLS database environment, build the MySQL docker image.
+
+```bash
+cd $SRC_BASE_DIR/yourls-action/yourls-mysql
+docker build -t yourls-mysql .
+```
+
+Start the MySQL database:
+```bash
+docker run -d \
+   -p 3306:3306 \
+   --name mysql \
+   -e "MYSQL_ROOT_PASSWORD=amazingpassword" \
+   yourls-mysql
+```
+
+### Create a namespace
+
+Create a namespace filesystem, and a CSV mapping file [links.csv](./links.csv).
```

Comment:
Is it going to be clear to a user what a namespace filesystem is if so disregard but I would be like 'whaaat'